frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -162,6 +178,74 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (!isWindows && windowsTools) {
     dispatch(vmWizardInternalActions[InternalActionType.RemoveStorage](id, windowsTools.id));
   }
+
+  // base image is always off for user templates
+  if (iGetVmSettingValue(state, id, VMSettingsField.USER_TEMPLATE)) {
```

Comment:
we can't do early return here because it will not reset the values when you select a template

Reply:
removed